code/html/index.html

```diff
@@ -1890,6 +1890,10 @@ <h2>
                 <div class="pure-u-1 pure-u-lg-1-4"><label>Pulse time (s)</label></div>
                 <div class="pure-u-1 pure-u-lg-1-4"><input name="relayTime" class="pure-u-1" type="number" min="0" step="0.1" max="3600" /></div>
             </div>
+            <div class="p-g module module-sch">
+                <div class="p-u-1 p-u-lg-1-4"><label>Restore last schedule</label></div>
```

Comment:
Note that this should probably be `pure-g`, gulp will shorten it afterwards.
Opening things locally breaks styling.

Since everything else is done, I'd rebuild the webui files and add them here + this fix.